### ISSUE TITLE
Add support for grouped requests

### ISSUE THREAD

**Issue by ghost** [iron-io/issues#3]
We are calculating an aggregate score based on a large number of cached individual scores. It would be very helpful to query, say, 50 keys at once, as opposed to sending 50 separate requests.

**Comment by treeder** [comment]
A multi get function is definitely on our roadmap, will update here when it's done.

**Comment by ghost** [comment]
Thank you for your reply. This is the only issue that keeps us from using IronCache.

**Comment by EpicDraws** [comment]
+1. This is a blocking issue for us as well.

**Comment by treeder** [comment]
Thanks guys, we're looking into this. 

Just to be clear in terms of usage, you'd want to pass in a list of specific keys and get the results in one big json array?

**Comment by EpicDraws** [comment]
Yes, for example a get which takes an array, and returns a hash. Also, Dalli for memcached has a multi block which allows for multiple set, update or add commands and batches them at the end of the block.
